nil/services/nilservice/config.go

```diff
@@ -43,6 +43,9 @@ type Config struct {
 	RPCPort        int                   `yaml:"rpcPort,omitempty"`
 	BootstrapPeers network.AddrInfoSlice `yaml:"bootstrapPeers,omitempty"`
 
+	// Profiling
+	PprofPort int `yaml:"pprofPort,omitempty"`
```

Comment:
you may also need to change `devnet.go` file

Reply:
Done.